kf_lib_data_ingest/etl/stage_analyses.py

```diff
@@ -27,12 +28,13 @@ def check_counts(discovery_sources, expected_counts):
     messages = ['UNIQUE COUNTS:\n' + pformat(uniques)]
     passed = True
 
+    # check expected counts
     if not expected_counts:
-        messages.append("No expected counts registered. ❌")
+        messages.append("No expected counts registered. ⚠️")
         passed = True  # Pass if we have no expectations
     else:
         checks = pandas.DataFrame(
-            columns=['Key', 'Expected', 'Found', 'Errors']
+            columns=['Key', 'Expected', 'Found', 'Equal']
```

Comment:
I really liked the way we had it before, I thought it was much more user friendly and easy to see quickly what went right/wrong

Reply:
everything that's not wrong is right, though